service/create/service.go

```diff
@@ -298,10 +313,11 @@ type runMachinesInput struct {
 	prefix              string
 }
 
-func (s *Service) runMachines(input runMachinesInput) error {
+func (s *Service) runMachines(input runMachinesInput) ([]*string, error) {
```

Comment:
The reason is that we feed these return values back into the AWS API (in this case, we pass the master's IP to the `WaitUntilInstanceRunning` method), and they always want `*string`s, so either we make the conversion each time we need to, or we just pass around `*string`s.